x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -7279,20 +7279,18 @@
     "xpack.enterpriseSearch.appSearch.productName": "App Search",
     "xpack.enterpriseSearch.appSearch.result.hideAdditionalFields": "追加フィールドを非表示",
     "xpack.enterpriseSearch.appSearch.roleMappings.title": "ロールマッピング",
-    "xpack.enterpriseSearch.appSearch.settings.logRetention.analytics.customPolicy": "カスタム分析保持ポリシーがあります。",
-    "xpack.enterpriseSearch.appSearch.settings.logRetention.analytics.ilmDisabled": "App Search は分析保持を管理していません。",
+    "xpack.enterpriseSearch.appSearch.logRetention.customPolicy": "カスタム {logsType} ログ保持ポリシーがあります。",
```

Comment:
Just curious, where are these translations coming from?

Reply:
There's an i18n team/contractors that I think check for untranslated IDs every release? I'm not super clear myself lol, the #kibana-localization channel might know. Also https://github.com/elastic/kibana/tree/master/packages/kbn-i18n for reference